### PR TITLE
Fix: Packaging added to question type dropdown

### PR DESCRIPTION
**Pull request by psankhe28**
### What
I tried to solve the issue https://github.com/openfoodfacts/hunger-games/issues/503

### Screenshot
Screenshot 2023-02-06 125234

The dropdown is working perfectly fine but I am unable to change the display name. It should be *packaging* instead of questions.packaging.

Can anyone help me with this?

### CODE REVIEW

File: src/pages/questions/QuestionDisplay.jsx [hunk 1]
```diff
@@ -321,28 +354,23 @@ const QuestionDisplay = ({
...
-          })
-        }
-        color="secondary"
-        variant="contained"
-        size="medium"
-        autoFocus
-        sx={{ py: "1rem" }}
-      >
-        {t("questions.skip")} ({shortcuts.skip})
-      </Button>
```
Comment (alexfauquette) [inline src/pages/questions/QuestionDisplay.jsx:344]: I don't get how this modification is related to the issue you try to fix smae for others diff which have no link with packaging

Comment (psankhe28) [reply src/pages/questions/QuestionDisplay.jsx:344]: I mistakenly added the files which were previously edited for some other issue. I will rectify it as soon as possible. Sorry for the inconvenience.

**Comment by alexfauquette** [comment]
@psankhe28 Do you need some help for git branch management?

**Comment by psankhe28** [comment]
> @psankhe28 Do you need some help for git branch management?

Will it be fine if I open a new pull request as I am getting confused between the codes?

**Comment by alexfauquette** [comment]
Every thing is fine, but if you want some help to improve your git workflow, it's also possible

**Pull request closed without merging**